terraform.tfvars.j2

```diff
@@ -0,0 +1,29 @@
+prefix = "{{ cluster_name }}"
+
+ansible_control_vm_flavor = "general.v1.small"
+ansible_control_vm_name   = "ansible-control"
+ansible_control_disk_size = 25
+
+seed_vm_flavor = "general.v1.small"
+seed_disk_size = 25
+
+multinode_flavor     = "general.v1.medium"
+multinode_image = "{{ multinode_image }}"
+multinode_keypair    = "MaxMNKP"
+multinode_vm_network = "stackhpc-ipv4-geneve"
+multinode_vm_subnet  = "stackhpc-ipv4-geneve-subnet"
+compute_count        = "2"
+controller_count     = "3"
+compute_disk_size    = 25
+controller_disk_size = 25
```

Comment:
This is unlikely to be large enough for a real deployment.

Reply:
Yes, I agree. This is just for the deployment of the VMs without OpenStack so it seemed unneccessary for me to request such large volumes for testing to see whether the App was capable of deploying VMs. The storage config is reverted for the branch which is concerned about deploying multinodes with OpenStack, however, these volumes will soon be a user defined input option.